src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -81,13 +85,23 @@ private void bindListeners(ReadOnlyPerson person) {
         phone.textProperty().bind(Bindings.convert(person.phoneProperty()));
         address.textProperty().bind(Bindings.convert(person.addressProperty()));
         email.textProperty().bind(Bindings.convert(person.emailProperty()));
+        initFav(person);
         person.tagProperty().addListener((observable, oldValue, newValue) -> {
             tags.getChildren().clear();
             initTags(person);
         });
     }
 
     /**
+     * Changes the star metaphor image for each {@code Person} according to their favorite status
+     */
+    private void initFav(ReadOnlyPerson person) {
+        if (person.getFavorite().toString().equals(Favorite.FavoriteStatus.no.toString())) {
```

Comment:
For instance, if we change Favorite's value to a boolean, we can just to if (person.getFavorite().isFavorite()) instead of checking for string equality (which runs in O(n) time!)

Reply:
Implemented and updated my PR! 👍